src/components/playerdash/player-card.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Firebase from 'firebase';
+import FirebaseLib from '../../utils/FirebaseLib.js';
+import utils from '../../utils/utilities.js';
+
+
+import { Link } from 'react-router';
+
+module.exports = React.createClass( {
+
+  componentWillMount() {
+    this.firebase = new FirebaseLib();
+  },
+
+  render() {
+    const { name, image, league, score, topScore, wins, losses,
+                id, bottomScore, streak, bestStreak, worstStreak} = this.props.player;
+
+    return (
+        <div className="panel panel-default">
+          <table className="table">
+            <thead>
+              <tr>
+                <th><img src={image} className="img-circle img-thumbnail" />{name} - {league} League</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
```

Comment:
These could be good to extract to a sub-component as there's lots of repetition.

```jsx
<tr>
  <td>Best Rating</td>
  <td>{topScore}</td>
</tr>
```
could become 

```jsx
<KeyValueRow key="Rating" value={score} />
```